errors.go.tmpl

```diff
@@ -58,6 +58,10 @@ func (e WebRPCError) WithCause(cause error) WebRPCError {
 	return err
 }
 
+func (e WebRPCError) WithCausef(msg string, args ...any) WebRPCError {
```

Comment:
Pls update the signature as in as in `fmt.Errorf()`:
```suggestion
func (e WebRPCError) WithCausef(fmt string, a ...interface{) WebRPCError {
```

Pls don't use word `any` for now. This template still works with Go 1.16+ as of today. Let's not break backward compatibility for a single line :)